src/test/java/persistence/sql/QueryTest.java

```diff
@@ -0,0 +1,114 @@
+package persistence.sql;
+
+import database.DatabaseServer;
+import database.H2;
+import jdbc.JdbcTemplate;
+import org.junit.jupiter.api.*;
+import persistence.sql.ddl.DDLQueryBuilder;
+import persistence.sql.dialect.Dialect;
+import persistence.sql.dialect.H2Dialect;
+import persistence.sql.dml.DMLQueryBuilder;
+import persistence.sql.model.Table;
+import persistence.study.sql.ddl.Person3;
+import persistence.study.sql.ddl.Person3RowMapper;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class QueryTest {
+
+    private static DatabaseServer server;
+    private static JdbcTemplate jdbcTemplate;
+    private static DDLQueryBuilder ddlQueryBuilder;
+    private static DMLQueryBuilder dmlQueryBuilder;
+
+    @BeforeAll
+    static void initialize() throws SQLException {
+        server = new H2();
+        server.start();
+
+        Connection connection = server.getConnection();
+        jdbcTemplate = new JdbcTemplate(connection);
+
+        Dialect dialect = new H2Dialect();
+        Table table = new Table(Person3.class);
+
+        ddlQueryBuilder = new DDLQueryBuilder(table, dialect);
+        dmlQueryBuilder = new DMLQueryBuilder(table);
+    }
+
+    @AfterAll
+    static void close() {
+        server.stop();
+    }
+
+    @BeforeEach
+    void setUp() throws SQLException {
+
+
+
+        String createTableQuery = ddlQueryBuilder.buildCreateQuery();
+        jdbcTemplate.execute(createTableQuery);
+
+        Stream<Person3> persons = createPersons();
+
+        persons.forEach(person -> {
+            String insertQuery = dmlQueryBuilder.buildInsertQuery(person);
+            jdbcTemplate.execute(insertQuery);
+        });
+    }
```

Comment:
```suggestion
    @BeforeEach
    void setUp() throws SQLException {
        String createTableQuery = ddlQueryBuilder.buildCreateQuery();
        jdbcTemplate.execute(createTableQuery);

        Stream<Person3> persons = createPersons();

        persons.forEach(person -> {
            String insertQuery = dmlQueryBuilder.buildInsertQuery(person);
            jdbcTemplate.execute(insertQuery);
        });
    }
```
불필요한 공백을 줄여보는것도 좋을 것 같아요 😃

Reply:
코드를 변경하다가 포맷팅은 신경쓰지 못했네요!
다음부터는 주의하도록 하겠습니다